frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.tsx

```diff
@@ -42,7 +59,7 @@ export const EmbedHomepage = () => {
       exampleDashboardId={exampleDashboardId}
       embeddingAutoEnabled={embeddingAutoEnabled}
       licenseActiveAtSetup={licenseActiveAtSetup}
-      plan="oss-starter"
+      defaultTab={defaultTab}
```

Comment:
I changed the prop here, I think the extra abstraction wasn't worth it, the plan was only used to determine the default tab